### PR TITLE
Goreport cleanup

### PR DESCRIPTION
**Pull request by trentis**
Cleaning up the goreport card a little

### CODE REVIEW

File: pkg/model/state-validate.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 )
 
+// ValidateState...
 type ValidateState struct {
```
Comment (alankm) [inline pkg/model/state-validate.go:9]: ...

File: pkg/model/state-validate.go [hunk 2]
```diff
@@ -13,6 +14,7 @@ type ValidateState struct {
 	Transition  string      `yaml:"transition,omitempty"`
 }
 
+// GetID...
```
Comment (alankm) [inline pkg/model/state-validate.go:17]: ...

File: pkg/model/state-validate.go [hunk 3]
```diff
@@ -32,6 +34,7 @@ func (o *ValidateState) getTransitions() map[string]string {
 	return transitions
 }
 
+// GetTransitions...
```
Comment (alankm) [inline pkg/model/state-validate.go:37]: ...

File: pkg/model/state-validate.go [hunk 4]
```diff
@@ -47,6 +50,7 @@ func (o *ValidateState) GetTransitions() []string {
 	return transitions
 }
 
+// Validate...
```
Comment (alankm) [inline pkg/model/state-validate.go:53]: ...

File: pkg/metrics/types.go [hunk 5]
```diff
@@ -2,24 +2,32 @@ package metrics
...
+	// NextRetry the state has retried
+	NextRetry // State retried
 )
 
+// NextEnums an array of all the NextEnum
 var NextEnums = []NextEnum{
 	NextEnd, NextTransition, NextRetry,
 }
 
+// InvokerEnum...
```
Comment (jalfvort) [inline pkg/metrics/types.go:22]: fix

File: pkg/metrics/types.go [hunk 6]
```diff
@@ -2,24 +2,32 @@ package metrics
...
 type InvokerEnum int
 
 const (
+	// InvokerUnknown...
```
Comment (jalfvort) [inline pkg/metrics/types.go:26]: fix

File: pkg/metrics/types.go [hunk 7]
```diff
@@ -2,24 +2,32 @@ package metrics
...
 	InvokerUnknown InvokerEnum = iota
 )
 
+// InvokerEnumLabels...
```
Comment (jalfvort) [inline pkg/metrics/types.go:30]: fix

**Review by jalfvort — COMMENTED** [review]
Also make sure to run test suite atleast once

**Review by jalfvort — APPROVED** [review]

**Pull request merged**